contracts/interfaces/external/IWETH.sol

```diff
@@ -1,7 +1,12 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity ^0.8.0;
 
-interface IWETH {
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
```

Comment:
use the '../interfaces/external/IERC20.sol' interface instead of openzeppelin

Reply:
ok

Reply:
ok